cmd/notation/logout.go

```diff
@@ -16,8 +16,9 @@ func logoutCommand(opts *logoutOpts) *cobra.Command {
 		opts = &logoutOpts{}
 	}
 	return &cobra.Command{
-		Use:   "logout [server]",
+		Use:   "logout [flags] [server]",
```

Comment:
```suggestion
		Use:   "logout [flags] <server>",
```
This doesn't look very correct to me. If server is optional, what happens when `notation logout` is called?

Reply:
Updated code and spec.